core/src/main/java/org/elasticsearch/indices/query/IndicesQueriesRegistry.java

```diff
@@ -19,18 +19,21 @@
 
 package org.elasticsearch.indices.query;
 
-import java.util.Map;
-
 import org.elasticsearch.common.component.AbstractComponent;
 import org.elasticsearch.common.settings.Settings;
 import org.elasticsearch.index.query.QueryParser;
+import org.elasticsearch.search.suggest.Suggesters;
+
+import java.util.Map;
 
 public class IndicesQueriesRegistry extends AbstractComponent {
     private Map<String, QueryParser<?>> queryParsers;
+    private Suggesters suggesters;
 
-    public IndicesQueriesRegistry(Settings settings, Map<String, QueryParser<?>> queryParsers) {
+    public IndicesQueriesRegistry(Settings settings, Map<String, QueryParser<?>> queryParsers, Suggesters suggesters) {
```

Comment:
I'm not sure we should add suggesters to this class as it means that anywhere we need to use queries (percolator, aggs, etc.) we need to have suggesters too. I think the suggesters should have their own registry class instead